workspaces/3scale/plugins/3scale-backend/catalog-info.yaml

```diff
@@ -0,0 +1,12 @@
+# https://backstage.io/docs/features/software-catalog/descriptor-format#kind-component
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: backstage-community-3scale-backend
+  title: '@backstage-community/plugin-3scale-backend'
+  description: The 3scale Backstage provider plugin synchronizes the 3scale content into the Backstage catalog.
+spec:
```

Comment:
Should we add the missing annotations to these plugins? The ACR plugin has the following:

```  
annotations:
    github.com/project-slug: backstage/community-plugins
    github.com/team-slug: backstage/community-plugins-maintainers
    sonarqube.org/project-key: backstage-community_plugins
```

Reply:
True.

Maybe it's better to remove generic `github.comteam-slug`s like `backstage/community-plugins-maintainers` and `backstage/community-plugin-owners`, also if they exists, but for the plugins they I've updated here, they doesn't match these plugins...

Reply:
All removed